### ISSUE TITLE
Replace Middleman with 11ty

### ISSUE THREAD

**Issue by ndorfin** [ndorfin/SouthernSunQueenBees#45]
This is a major piece of work, and would require weeks of time.

What do you think @nickmangos?

### Advantages

- Uses a better supported SSG (11ty.dev)
- Easier to operate for JS-capable devs
- Familiar to me (@ndorfin)
- Quicker builds
- Quicker to debug

### Disadvantages

- The current site is working; we'd introduce risk in migrating to a new platform.
- We'd introduce new bugs
- We'd have to run several audits to make sure we're not missing anything.
- A whole new system to learn (for folks like Nick @nickmangos)

### Items in our favour

- We could feasibly re-use most of the assets, and HTML, as is.
- We could move to JSON for our content, meaning we could ditch the data transforms for building the YAML. We might be able to use the Contentful data directly!

### To do

 - [ ] Work in a new branch, locally only.
 - [ ] Replace Middleman (Ruby) infrastructure with Eleventy (Node). Gems to Packages. etc.
 - [ ] Move to JSON for contentful data
 - [ ] Use Contentful Node packages
 - [ ] Replace ERBs with Nunjucks
 - [ ] Rework layout system
 - [ ] Replace SCSS with normal CSS
 - [ ] Introduce bundling for JS and CSS
 - [ ] Rewrite build workflows to cater for Node, rather than Ruby

**Comment by nickmangos** [comment]
I don't think it has good ROI unless you were doing it an an exercise to build a template for future websites. But what do I know?
